libraries/AP_HAL_ChibiOS/hwdef/BETAFPV-F405-I2C/hwdef.dat

```diff
@@ -14,9 +14,11 @@ define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_GPS
 I2C_ORDER I2C2
 PB10 I2C2_SCL I2C2
 PB11 I2C2_SDA I2C2
-PB4 EXTERN_GPIO1 OUTPUT GPIO(82)
+PB4 EXTERN_GPIO1 OUTPUT GPIO(81)
 undef RELAY2_PIN_DEFAULT
-define RELAY2_PIN_DEFAULT 82
+define RELAY2_PIN_DEFAULT 81
+
+PC6 USART6_TX USART6
```

Comment:
can a comment be added to explain what USART6 is for?
also, why does the i2c variant need DMA on this UART when the base board doesn't?
also, how does this PR work? The base board BETAFPV-F405 doesn't have a 'NODMA' flag on USART6, so this seems to be a no-change PR apart from the GPIO numbering?

Reply:
The key thing is removing the ALT config - it seems that this messes with the pin configuration which means the GPS never configures correctly. So you have to remove the TX pin and then add it back in again to get rid of the ALT config

Reply:
Comment added